artifactory/commands/transferfiles/utils.go

```diff
@@ -350,7 +340,7 @@ func uploadChunkWhenPossibleHandler(phaseBase *phaseBase, chunk UploadChunk, upl
 		shouldStop := uploadChunkWhenPossible(phaseBase, chunk, uploadTokensChan, errorsChannelMng)
 		if shouldStop {
 			// The specific error that triggered the stop is already in the errors channel
-			return errorutils.CheckErrorf("%s stopped.", logMsgPrefix)
+			return errorutils.CheckErrorf("%sstopped.", logMsgPrefix)
```

Comment:
Unless this white-space was removed intentionally, let's add it back.

Reply:
This is intentional - there is already a space in the log prefix:
https://github.com/jfrog/jfrog-client-go/blob/v1.22.0/utils/utils.go#L309